dist/index.html

```diff
@@ -20,30 +26,34 @@
         <nav class="main-header__nav">
           <div class="nav__navigation left">
             <span class="logo">
-              FITNESS
+              <a href="#">FITNE<span class="orange">SS</span></a>
             </span>
-            <ul class="nav-links">
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
+            <ul class="nav-links hide">
+              <li><a href="#">HOME</a></li>
+              <li><a href="#">ABOUT US</a></li>
+              <li><a href="#">PAGE</a></li>
+              <li><a href="#">BLOG</a></li>
+              <li><a href="#">CONTACTS</a></li>
             </ul>
           </div>
-          <div class="nav__login right">
+          <div class="nav__login right hide">
```

Comment:
[same comment as this one](https://github.com/m3ntorship/fitness/pull/11/files#r407261368)